### PR TITLE
Add countdown timer Icon

### PR DESCRIPTION
**Pull request by adinc-transporeon**
## Description

References #1690

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: stencil-workspace/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@trimble-oss/modus-web-components",
-  "version": "0.1.53",
+  "version": "0.1.54",
```
Comment (msankaran0712) [inline stencil-workspace/package.json:3]: this one we can take care.

cc @coliff

Comment (coliff) [reply stencil-workspace/package.json:3]: we don't recommend users to make a PR bumping the version number in `package.json`, but in this instance I think it's fine.

File: stencil-workspace/src/components/icons/IconMap.tsx [hunk 2]
```diff
@@ -89,6 +90,8 @@ export const IconMap: FunctionalComponent<IconMapProps> = (props: IconMapProps)
       return <IconCollapseAll color={props.color} onClick={props.onClick} size={props.size} />;
     case 'copy':
       return <IconCopy color={props.color} onClick={props.onClick} size={props.size} />;
+    case 'countdown-timer':
+      return <IconCountdownTimer color={props.color} onClick={props.onClick} size={props.size} />;
```
Comment (msankaran0712) [inline stencil-workspace/src/components/icons/IconMap.tsx:94]: I would recommend renaming it to `timer-countdown` to be in sync with Modus Icons

**Review by msankaran0712 — CHANGES_REQUESTED** [review]

**Review by coliff — DISMISSED** [review]

File: stencil-workspace/src/components/icons/IconMap.tsx (hunk 2 above)
Comment (adinc-transporeon) [reply stencil-workspace/src/components/icons/IconMap.tsx:94]: adapted

**Review by msankaran0712 — APPROVED** [review]

**Pull request merged**